docs/examples/index.rst

```diff
@@ -58,24 +58,18 @@ Machine Translation
 .. container:: cards
 
    .. card::
-      :title: Google Neural Machine Translation
+      :title: Google Neural Machine Translation with Transformer
```

Comment:
GNMT and Transformers are two different models so the proposed title doesn't make sense.

Reply:
Thanks for your advice. I have made changes accordingly.